.circleci/config.yml

```diff
@@ -10,7 +10,14 @@ jobs:
       - run:
           name: "Build COMPASS docker image"
           command: |
-            docker build . -f docker/Dockerfile
+            docker build . -f docker/Dockerfile -t test_image
+      - run:
+          name: "Run tests"
+          command: |
+            docker run --entrypoint "cd ~/OPERA/COMPASS &&\
```

Comment:
`"docker run" requires at least 1 argument.`

The currnt circle CI build is failing from this. I think there might be a simpler way to run the tests, lemme check

Reply:
` docker run -t opera-adt/compass cd /home/compass_user/OPERA/COMPASS && conda init bash && conda activate COMPASS && python -m pytest` also fails after building

Reply:
@LiangJYu any opinion about this?

Reply:
Addressed in [e033b2e](https://github.com/opera-adt/COMPASS/pull/80/commits/e033b2e625710965a486ef9aad4fa07639e197bd)